fjord/search/index.py

```diff
@@ -41,34 +120,23 @@ def es_analyze(text, analyzer=None):
     return ret['tokens']
 
 
-def register_mapping_type(mapping_type):
-    """Registers a mapping type.
+def get_doctypes(doctypes=None):
+    """Returns dict of name -> DocType subclass for given doctype names
 
-    This gives us a way to get all the registered mapping types for
-    indexing.
-
-    """
-    _mapping_types[mapping_type.get_mapping_type_name()] = mapping_type
-    # Enable this to be used as a decorator
-    return mapping_type
+    :arg doctypes: list of doctype names to return
 
-
-def get_mapping_types(mapping_types=None):
-    """Returns a dict of name -> mapping type.
-
-    :arg mapping_types: list of mapping type names to restrict
-        the dict to.
+    :returns: dict of name -> DocType subclass
 
     """
-    if mapping_types is None:
-        return _mapping_types
+    if doctypes is None:
+        return _doctypes
 
-    return dict((key, val) for key, val in _mapping_types.items()
-                if key in mapping_types)
+    return dict((key, val) for key, val in _doctypes.itesm()
+                if key in doctypes)
 
 
-def get_index():
-    """Returns the index we're using."""
+def get_index_name():
```

Comment:
I renamed this because elasticsearch-dsl has an Index class and I decided it was worth clarifying this function.